js/backup_restore.js

```diff
@@ -566,7 +566,10 @@ function configuration_restore(callback) {
                     mincheck:               1100,
                     maxcheck:               1900,
                     rx_min_usec:            885,
-                    rx_max_usec:            2115
+                    rx_max_usec:            2115,
```

Comment:
Just going with the flow here. The configurator is not the nicest bit of code I've seen...